index.html

```diff
@@ -42,7 +42,7 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
             <!-- the first card............... -->
             <section id="work-section">
                 <div class="cards">
-                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <img class="card-img"  src="img2/Snapshoot Portfolio.png" alt="portfolio img">
                     <div class="align-text">
                     <h2 class="h2-size">Tonic</h2>
```

Comment:
- [ ] [Optional] I would suggest you please use the `background: #FFFFFF;` for your cards to match the Figma design. Implement with all the cards
### your design
![hos](https://user-images.githubusercontent.com/81727988/236515587-5bb3eb91-3b6f-417a-b03b-2fe105c63f56.png)

### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/236515617-ad4d2c2d-f5ed-48d4-b5f9-2bda4782dbf7.png)